game/game.go

```diff
@@ -8,10 +8,10 @@ type Game struct {
 	Stories []*Game
 }
 
-func NewGame(n string) *Game {
+func NewGame(n string) Game {
 	lastId++
```

Comment:
We should remove this global var in subsequent MRs

Reply:
:+1: